.github/workflows/tests.yml

```diff
@@ -7,19 +7,24 @@ on:
       - master
     tags:
       - '*'
+    paths-ignore:
+      - '**/*.md'
+      - 'LICENSE'
+      - 'Dockerfile'
+      - 'docker-compose.yml'
+      - 'dc.build.node.yml'
```

Comment:
will not test if only markdown type files or other non-relevant files are changed